OPUNetTransportLayer.cpp

```diff
@@ -626,20 +604,13 @@ int OPUNetTransportLayer::Send(Packet* packet)
 
 int OPUNetTransportLayer::Receive(Packet* packet)
 {
-	int i;
-	unsigned long numBytes;
-	sockaddr_in from;
-	bool bRetVal;
-	int sourcePlayerNetID;
-	int expectedPlayerNetID;
-
 	for (;;)

```

Comment:
Should this become a while loop?  Can be moved to an issue as it isn't directly tied to these changes.